ucf-document.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/*
+Plugin Name: UCF Document Plugin
+Description: Provides a custom post type and taxonomies for managing documents
+Version: 0.1.0
+Author: UCF Web Communications
+License: GPL3
+GitHub Plugin URI: UCF/UCF-Document-Plugin
+*/
+
+namespace UCFDocument;
+use UCFDocument\Document;
```

Comment:
`use` needs to be updated to reflect the namespace change in your last commit

Reply:
Updated.